src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -58,10 +57,23 @@
     public static final String ADDRESS_DESC_BOB = " " + PREFIX_ADDRESS + VALID_ADDRESS_BOB;
     public static final String DELIVERY_DATE_DESC_AMY = " " + PREFIX_DELIVERY_DATE + VALID_DELIVERY_DATE_AMY;
     public static final String DELIVERY_DATE_DESC_BOB = " " + PREFIX_DELIVERY_DATE + VALID_DELIVERY_DATE_BOB;
-    public static final String STATUS_DESC_AMY = " " + PREFIX_STATUS + VALID_STATUS_AMY;
-    public static final String STATUS_DESC_BOB = " " + PREFIX_STATUS + VALID_STATUS_BOB;
-    public static final String TAG_DESC_FRIEND = " " + PREFIX_TAG + VALID_TAG_FRIEND;
-    public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
+
+    // list of status
+    public static final String VALID_STATUS_PENDING = "PENDING";
+    public static final String VALID_STATUS_DELIVERING = "DELIVERING";
+    public static final String VALID_STATUS_OVERDUE = "OVERDUE";
+    public static final String VALID_STATUS_COMPLETED = "COMPLETED";
+    public static final String STATUS_DESC_PENDING = " " + PREFIX_STATUS + VALID_STATUS_PENDING;
+    public static final String STATUS_DESC_DELIVERING = " " + PREFIX_STATUS + VALID_STATUS_DELIVERING;
+    public static final String STATUS_DESC_COMPLETED = " " + PREFIX_STATUS + VALID_STATUS_COMPLETED;
+
+    // list of tags
+    public static final String VALID_TAG_FROZEN = "FROZEN";
+    public static final String VALID_TAG_FRAGILE = "FRAGILE";
```

Comment:
Changing of tags should have been done as a separate PR, but it's fine this time

Reply:
yeap I agree, but I wanted to make testing easier for everyone, cuz now our test codes are all over the place. so I wanted to standardize it so easier for everyone to test, rather than randomly writing some numbers